index.html

```diff
@@ -96,6 +96,7 @@ <h1 class="center space">#BlackLivesMatter</h1>
       <li>Antwon Rose II</li>
       <li>Keith Childress</li>
       <li>Walter Scott</li>
+      <li>... Far too many to list</li>
```

Comment:
```suggestion
      <li>... And countless more</li>
```

Bikeshedding the wording should not be a blocker, just thinking out loud.